spec/migrations/use_encrypted_data_for_account_credentials_spec.rb.disabled

```diff
@@ -193,10 +194,12 @@ def decrypt(team_password)
   private
 
   def decrypt_attr(attr, team_password)
-    encrypted_value = encrypted_data[attr].try(:[], :data)
+    data = encrypted_data[attr].try(:[], :data)
+    iv = encrypted_data[attr].try(:[], :iv)
 
-    cleartext_value = if encrypted_value
-                        Crypto::Symmetric::Aes256.decrypt(encrypted_value, team_password)
+    encrypted_values = { data: data, iv: iv }
```

Comment:
Create additional method to retrieve `encrypted_values` to increase readability and to keep it DRY?

Reply:
Ignored as migration is disabled